RELEASE_NOTES

```diff
@@ -3,6 +3,32 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
+v2.3.0 (September 2nd 2015)
+---------------------------
+
+feature + bugfix release
+- requires vsc-base v2.2.4 or more recent (#1343)
+    - required for mk_rst_table function in vsc.utils.docs
+- various other enhancements, including:
+    - add support for generating documentation for (generic) easyblocks in .rst format (#1317)
```

Comment:
OK, many nice new features & bugfixes in this release, thanks for the above and all the other bits; 
release notes LGTM!